src/raster/rasterband.rs

```diff
@@ -662,9 +662,9 @@ impl<'a> RasterBand<'a> {
                 window.1 as c_int,
                 window_size.0 as c_int,
                 window_size.1 as c_int,
-                buffer.data.as_mut_ptr() as *mut c_void,
-                buffer.size.0 as c_int,
-                buffer.size.1 as c_int,
+                buffer.data_mut().as_mut_ptr() as *mut c_void,
+                shape.0 as c_int,
```

Comment:
Ditto

Reply:
Oh, sorry, I thought you'd already merged `master`.

Reply:
Actually I had, but hadn't addressed these cases yet. Will do.